recipe/conda_build_config.yaml

```diff
@@ -11,11 +11,5 @@ python:        # [win]
   - 3.6        # [win]
   - 3.7        # [win]
 
-c_compiler:    # [osx]
-  - llvm       # [osx]
-  - clang      # [osx]
-cxx_compiler:  # [osx]
-  - llvm       # [osx]
-  - clangxx    # [osx]
-llvm:          # [osx]
-  - 4.0        # [osx]
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - '10.12'                # [osx]
```

Comment:
The llvm version in the pinning before should probably be higher than 4.0 (6.0 or 7.0 maybe, I don't know), but this new change here looks better, specially if conda adds a virtual package for this target :)